staging/info_box.md

```
@@ -0,0 +1,70 @@
+done | info_box
+
+# Static message
+info_box "Operation complete"
+
+# Show help
+info_box --help
+```
+
+#### Help Output
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)